src/lib.rs

```diff
@@ -348,6 +363,36 @@ impl<'text> BidiInfo<'text> {
         }
     }
 
+    // /// Returns if the paragraph is Left direction, right direction or mixed.
+    // pub fn get_direction(&self, para: &ParagraphInfo) -> Direction  {
```

Comment:
Please remove the commented-out code.

Reply:
(I think this is just part of their WIP work, it's a draft PR)

Reply:
👍